x/concentrated-liquidity/tick_test.go

```diff
@@ -263,3 +264,51 @@ func (s *KeeperTestSuite) TestInitOrUpdateTick() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestGetTickInfo() {
+	tests := []struct {
+		name             string
+		poolToGet        uint64
+		tickToGet        int64
+		expectedTickInfo model.TickInfo
+	}{
+		{
+			name:             "Get tick info on existing pool and existing tick",
+			poolToGet:        validPoolId,
+			tickToGet:        DefaultCurrTick.Int64(),
+			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg()},
+		},
+		{
+			name:             "Get tick info on existing pool with no existing tick",
+			poolToGet:        validPoolId,
+			tickToGet:        DefaultCurrTick.Int64() + 1,
+			expectedTickInfo: model.TickInfo{LiquidityGross: sdk.ZeroDec(), LiquidityNet: sdk.ZeroDec()},
+		},
+		{
+			name:             "Get tick info on a non-existing pool with no existing tick",
```

Comment:
Why do we not return an error in such a case instead? Doesn't make sense to return a valid tick info with zero values in such a case

Reply:
Okay, I added a poolExists check. Basically if the pool does not exist, it returns an error with nil values for tickInfo. If the pool exists but the ticks just arent initialized, it returns a valid tickInfo object but with zero values for liquidity